packages/botonic-nlu/README.md

```diff
@@ -255,17 +262,18 @@ It splits the loaded dataset in two sets: one for training the model and the oth
 
 ```ts
 trainTestSplit(options: {
-    data: DataSet;
-    testPercentage: number;
-    stratify?: boolean;
+  data: DataSet;
```

Comment:
@elozano98 @vanbasten17 why the reformatting? do you have pre-commit configured?

Reply:
i did it manually, it looks better once rendered the markdown